js/src/components/05_pages/Course/index.js

```diff
@@ -68,8 +68,9 @@ const StyledShadowBox = withStyles(theme => ({
 
 const StyledBox = withStyles(theme => ({
   root: {
-    marginTop: theme.spacing(2),
-    alignItems: 'baseline',
```

Comment:
Hi @ikorgik you need to keep the `alignItems: 'baseline'` - otherwise the button and text aren't vertically cetnred to each other.

Reply:
Done!